content/blog/spatialsample-0-2-0/index.Rmd

```diff
@@ -0,0 +1,205 @@
+---
+output: hugodown::hugo_document
+
+slug: spatialsample-0-2-0
+title: spatialsample 0.2.0
+date: 2022-06-15
+author: Mike Mahoney
+description: >
+    spatialsample 0.2.0 is now on CRAN! This release provides a bunch of new features, including new spatial resampling methods, visualization helpers, and spatial buffering.
+
+photo:
+  url: https://unsplash.com/photos/1-29wyvvLJA
+  author: Andrew Neel
+
+# one of: "deep-dive", "learn", "package", "programming", "roundup", or "other"
+categories: [package] 
+tags: [spatialsample, rsample, tidymodels]
+---
+
+<!--
+TODO:
+* [x] Look over / edit the post's title in the yaml
+* [x] Edit (or delete) the description; note this appears in the Twitter card
+* [x] Pick category and tags (see existing with `hugodown::tidy_show_meta()`)
+* [x] Find photo & update yaml metadata
+* [x] Create `thumbnail-sq.jpg`; height and width should be equal
+* [x] Create `thumbnail-wd.jpg`; width should be >5x height
+* [x] `hugodown::use_tidy_thumbnails()`
+* [x] Add intro sentence, e.g. the standard tagline for the package
+* [x] `usethis::use_tidy_thanks()`
+-->
+
+```{r setup, include = FALSE}
+library(ggplot2)
+theme_set(theme_minimal())
+```
+
+
+We're positively electrified to announce the release of [spatialsample](https://spatialsample.tidymodels.org/) 0.2.0. spatialsample is a package for spatial resampling, extending the rsample framework for resampling to help create spatial extrapolation between your analysis and assessment data sets.
+
+You can install it from CRAN with:
+
+```{r, eval = FALSE}
+install.packages("spatialsample")
+```
+
+This blog post will describe the highlights of what's new. You can see a full list of changes in the [release notes](https://github.com/tidymodels/spatialsample/blob/main/NEWS.md).
```

Comment:
it'd be nice to update this link to the `0.2.0` section in the NEWS of the pkgdown site, once it's published. then it goes to the right place permanently.

Reply:
Thanks!